2-Draft-Accepted/RFCNNNN-PSCore6-and-Windows-Modules.md

```diff
@@ -0,0 +1,83 @@
+---
+RFC: RFCNNNN
+Author: Steve Lee
+Status: Draft
+SupercededBy: n/a
+Version: 0.1
+Area: Engine
+Comments Due: 6/30
+Plan to implement: Yes
+---
+
+# Enable Discovery and Importing of Compatible in-box Windows PowerShell modules
+
+The biggest factor preventing existing Windows PowerShell users from moving to PowerShell Core 6 has been cmdlet coverage.
+Specifically being able to use existing cmdlets that ship with Windows.
+The PowerShell team has been working with Windows teams to port their modules so that they work in PowerShell Core 6.
+This RFC addresses the experience for the user when using PowerShell Core 6.1 with next version of Windows 10.
+
+## Motivation
+
+As an ITPro,
+I can use existing Windows PowerShell modules in PowerShell Core 6,
+so that I can deploy PowerShell Core 6 in my enterprise.
+
+## Specification
+
+The module manifest (since Windows PowerShell 5) may contain a `CompatiblePSEditions` property to indicate if it is compatible with
+Windows PowerShell (value of `Desktop` even on Windows Server) or PowerShell Core (value of `Core`).
+Currently, this property is only documentation and not observed by PowerShell.
+The proposal is to modify PowerShell Core 6.1 to respect this property for in-box Windows PowerShell modules.
+
+### Windows PSModulePath
+
+Windows PowerShell, by default, has module paths in:
+
+  1. User profile: $env:UserProfile\Documents\WindowsPowerShell\Modules
+  2. Program files: $env:ProgramFiles\WindowsPowerShell\Modules
+  3. System32: $env:Windir\System32\WindowsPowerShell\v1.0\Modules
+
+The proposed change is to add *only* the `System32` path to `PSModulePath` on PowerShell Core 6.
+
+Modules in the user profile should be re-installed under PowerShell Core 6.
```

Comment:
Why? Will it re-installed to another path?

Reply:
I can add more context here if needed, but the point is that if you run `install-module` from Windows PowerShell, it installs to the Windows PowerShell location and won't be found by PSCore6 (unless `Add-WindowsPSModulePath` is used)

Reply:
If a module was installed in `$home\Documents\WindowsPowerShell\Modules` where it will be after re-install?